### ISSUE TITLE
Won't stay signed into Google

### ISSUE THREAD

**Issue by reinbeau** [inbasic/ignotifier#356]
Seems to be unable to remain signed into Google - I think this happened two Firefox updates ago.  Any suggestions?

**Comment by inbasic** [comment]
I am not sure why! So the toolbar button switches back and forth between blue and (red/grey)?

**Comment by reinbeau** [comment]
Yes. I shut down Firefox last night, this morning I've just opened it up, and the icon is greyed out. When I click on it I have to sign into Google to view my Gmail account. Is this a new Firefox thing that they aren't cooperating with Google? The other thing, unrelated to your extension, is that every single time I get a Firefox upgrade the search engine is switched to Yahoo. I really think Mozilla is becoming too big for their britches....Thank you for any help you can offer. 

Take care, 

Ann 

----- Original Message -----

**Comment by inbasic** [comment]
> the icon is greyed out

The grey mean you are logged in but you dont have any new message in your inbox

> is that every single time I get a Firefox upgrade the search engine is switched to Yahoo. I really think Mozilla is becoming too big for their britches....Thank you for any help you can offer.

Seems like your Firefox profile is damaged. Try migrating to a new profile,
Read FAQ 10, http://add0n.com/gmail-notifier.html

**Comment by reinbeau** [comment]
Ok, I used the wrong term. It's 'blued'. Believe me, I've just opened up Firefox, and it's up there blued out, I'm going to have to sign in. Every single time. This wasn't the case before. It's irritating, not the end of the world, but it isn't working the way it used to. It will remain signed in for this session, then I have to do it again if I close and reopen Firefox. I've used your extension for years (and I've donated for it), so I do know how it is supposed to work. 

Take care, 

Ann 

----- Original Message -----
